util/src/misc.rs

```diff
@@ -69,5 +70,19 @@ pub fn contents(name: &str) -> Result<Bytes, UtilError> {
 
 /// Get the standard version string for this software.
 pub fn version() -> String {
-	format!("Parity//{}-{}-{}/{}-{}-{}/rustc{}", env!("CARGO_PKG_VERSION"), short_sha(), commit_date().replace("-", ""), Target::arch(), Target::os(), Target::env(), rustc_version::version())
+	format!("Parity//v{}-{}-{}/{}-{}-{}/rustc{}", env!("CARGO_PKG_VERSION"), short_sha(), commit_date().replace("-", ""), Target::arch(), Target::os(), Target::env(), rustc_version::version())
+}
+
+/// Get the standard version data for this software.
+pub fn version_data() -> Bytes {
```

Comment:
given the output is based on environmental factors outside the control of the test environment, and test would simply be a copy'n'paste of the code in the function.